veslog.md

```diff
@@ -153,16 +191,13 @@ If you need to extract many years of VESLOG data, do it in a loop.
 The following SQL code stitches together vtr records from GARFO's C,I,D tables
 ```
 select d.*, i.*, c.*
-  from noaa.document@garfo_nefsc d,  -- vessel permit, sail, land, crew, trip category (commercial, P/C, rec)
-       noaa.images@garfo_nefsc i,    -- Subtrip information, gear, effort, area
-       noaa.catch@garfo_nefsc c      -- Species, kept, discarded, who sold to, ddate_sold
+  from NEFSC_GARFO.TRIP_REPORTS_DOCUMENT d,  -- vessel permit, sail, land, crew, trip category (commercial, P/C, rec)
```

Comment:
This information conflicts with what is listed above, what is true souce CATCH or TRIP_REPORTS_CATCH

Reply:
[bdc4063](https://github.com/NEFSC/READ-SSB-Lee-metadata/pull/27/commits/bdc4063b34c903c7fe46bd65c26132b660e5fa1f)

Reply:
Added to #30. We'll update it when ITD sorts it out.

Reply:
Would suggest keeping this open or adding a note so it is not forgotten then we are suggesting a source which is not verified